src/client/client4.js

```diff
@@ -1081,10 +1081,10 @@ export default class Client4 {
         );
     };
 
-    addToChannel = async (userId, channelId) => {
+    addToChannel = async (userId, channelId, postRootId) => {
```

Comment:
Minor but can we default this to `''`? e.g. `... postRootId = '')` That way if it's not used we're not sending null value to the server